src/sentry/tagstore/snuba/backend.py

```diff
@@ -741,7 +742,7 @@ def get_tag_value_paginator_for_projects(
             conditions=conditions,
             orderby=order_by,
             # TODO: This means they can't actually paginate all TagValues.
-            limit=1000,
```

Comment:
If this query worked with 1000, please leave it to 1000. if it worked, it means there is no reason for potentially load 10x more data.